src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -77,8 +77,11 @@ public function evaluate($functions, $values)
                 if (!is_object($obj)) {
                     throw new \RuntimeException('Unable to get a property on a non-object.');
                 }
+                if (($toCall = array($obj, $this->nodes['attribute']->attributes['value'])) && !is_callable($toCall)) {
```

Comment:
`if (!is_callable($toCall = array($obj, $this->nodes['attribute']->attributes['value']))) {`

Reply:
@nicolas-grekas fixed